packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -188,20 +188,15 @@ export class TreeItem extends FASTElement {
         return true;
     };
 
-    public handleExpandCollapseButtonClick = (): void => {
+    public handleExpandCollapseButtonClick = (e: MouseEvent): void => {
         if (!this.disabled) {
+            e.preventDefault();
```

Comment:
Hypothetical question on this - lets say that a team passes our upcoming telemetry directive to a tree item and expects to measure whether an item was expanded or selected. Would this suppress the event bubbling to that capture?

Reply:
preventDefault does not change bubbling behavior, it just enables other event consumers to tell some other code has taken some action based on that event. That's why I think preventDefault is a better way to address this than stopPropagation.

Reply:
doh